src/Behat/Behat/DependencyInjection/config/behat.xml

```diff
@@ -305,8 +305,10 @@
                 <service class="%behat.translator.message_selector.class%" />
             </argument>
 
-            <call method="setFallbackLocale">
-                <argument>en</argument>
+            <call method="setFallbackLocales">
```

Comment:
no they are not. the Translation component defines dev requirements on them only, not requirements